bundle/manifests/kubernetes-nmstate-operator.clusterserviceversion.yaml

```
@@ -53,6 +53,28 @@ metadata:
                   "name": "br1",
                   "state": "up",
                   "type": "linux-bridge"
+                },
+                {
+                  "bridge": {
+                    "options": {
+                      "stp": {
+                        "enabled": false
+                      }
+                    },
+                    "port": [
+                      {
+                        "name": "eth3"
+                      }
+                    ]
+                  },
+                  "ipv4\"": {
```

seems pretty weird to have a key with a quote in it?
```suggestion
                  "ipv4": {
```

---

updated